### PR TITLE
[TR] Merge 1 3 into 1 4

### PR DESCRIPTION
**Pull request by jjanvier**

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/CompletenessGenerator.php [hunk 1]
```diff
@@ -140,7 +140,10 @@ public function getCompletenesses(array $normalizedData, array $normalizedReqs)
         $normalizedData = array_filter(
             $normalizedData,
             function ($value) {
-                return (null !== $value);
+                //TODO: should be dropped on master as we don't normalize anymore empty medias in the normalized data
+                $nullNormalizedMedia = ['filename' => null, 'originalFilename' => null];
+
+                return (null !== $value && $nullNormalizedMedia !== $value);
```
Comment (ClementGautier) [inline src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/CompletenessGenerator.php]: why don't you does:

```php
return null !== $value && null !== $value['filename'] && null !== $value['originalFilename'];
```

**Comment by ClementGautier** [comment]
a little comment but as it's not your code (I assume its only a merge PR) GTM even if you don't want to change it.

**Comment by jjanvier** [comment]
it was my code from the 1.3 fix

I agree with you, the var was just here to make understand this hackish fix

**Comment by nidup** [comment]
GTM

**Pull request merged**